pom.xml

```diff
@@ -50,7 +51,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>github-api</artifactId>
-            <version>1.71</version>
+            <version>1.72</version>
```

Comment:
Not really necessary, but somehow @kohsuke seems to have published 1.71 without a `*-sources.jar`, making it harder to debug issues.